include/sys/statvfs.h

```diff
@@ -5,26 +5,44 @@
  *
  * sys/statvfs.h
  *
- * Copyright 2018 Phoenix Systems
- * Author: Michał Mirosław
+ * Copyright 2022 Phoenix Systems
+ * Author: Lukasz Kosinski
  *
  * This file is part of Phoenix-RTOS.
  *
  * %LICENSE%
  */
 
-#ifndef _LIBPHOENIX_SYS_STATVFS_H
-#define _LIBPHOENIX_SYS_STATVFS_H
+#ifndef _LIBPHOENIX_SYS_STATVFS_H_
+#define _LIBPHOENIX_SYS_STATVFS_H_
 
+#include <sys/types.h>
 
-struct statfs
-{
+
+/* Filesystem flags */
+enum {
+	ST_RDONLY = (1 << 0), /* Read-only filesystem */
+	ST_NOSUID = (1 << 1)  /* Ignore S_ISUID and S_ISGID file mode bits */
```

Comment:
I think these should be defines (also using enum as int or as bitmask is a no-no in MISRA)

Reply:
Fixed, I've changed the enum into defines.